### ISSUE TITLE
Dream's Herald can't be sold in AH

### ISSUE THREAD

**Issue by Supergirl1A** [slowtorta/turtlewow-bug-tracker#4280]
**Please, provide a clear description what the bug is:**
the Dream's Herald I crafted can't be sold in AH.It showed a mana made equipment like oranges.Why isn't the crafted weapon like Sulfuron Hammer or Nightfall.
With GM's help,I have deleted my WDB but it's still not effective.
**Steps to reproduce the behavior with as much detail as possible:**

1.
2. 
3. 

**IDs of affected creatures, items, quests or spells with a link to the relevant page:**

NPC: https://database.turtle-wow.org/?npc=
Item: https://database.turtle-wow.org/?item=65008
Quest: https://database.turtle-wow.org/?quest=
Spell: https://database.turtle-wow.org/?spell=
Object: https://database.turtle-wow.org/?object=

**Expected behavior. Describe how it should work:**
Is this a BUG or game setting?
It's a great inconvenience for blacksmith crafter.
Hope u check it and reply,thanks.

**Comment by Supergirl1A** [comment]
Please see my friend respond to me.

**Comment by Supergirl1A** [comment]
Can developers solve this exception?

**Comment by Supergirl1A** [comment]
> It's a bug, dreams herald wasn't crated properly. And first it didn't even have a durability value. Must be something still not set right in the SQL database

Yes! Yes! This must be abnormal! This has been bothering me for a long time! I don't know why this problem persists until now and cannot be solved!

**Comment by balakethelock** [comment]
> Can developers solve this exception?

yes they can
@Dragunovi

**Comment by Supergirl1A** [comment]
@Dragunovi  Hi！Can you  help me ???

**Comment by twowTurtReynolds** [comment]
![image](https://github.com/slowtorta/turtlewow-bug-tracker/assets/151180101/899c7778-3a6e-45bd-9164-ab74b0ce8ef8)

This is odd. The crafting spell must have been a bad choice to copy. When crafting it also uses the sounds+anims of leatherworking crafting, not of banging on anvil.
